src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -984,7 +984,7 @@ public static boolean isArray(final Object object) {
 
     /**
      * Checks if an Object is empty or null.
-     *
+     * <p></p>
```

Comment:
Uh?

Reply:
`<p>`This is a paragraph.`</p>`

Reply:
done

Reply:
Did you not see my example?

Reply:
Are you not trying a local build before you push to git?

Reply:
@arturobernalg 
`<p>`This is a paragraph.`</p>`, you can't embed lists per the errors in the build. Again, run a 'mvn' build before you push and you will discover this type of issue without forcing reviewers to ping you for broken builds. TY.